IPython/utils/traitlets.py

```diff
@@ -1635,14 +1635,14 @@ def validate_elements(self, obj, value):
 class Dict(Instance):
     """An instance of a Python dict."""
 
-    def __init__(self, default_value=None, allow_none=True, **metadata):
+    def __init__(self, default_value={}, allow_none=True, **metadata):
```

Comment:
Won't this will use the same dictionary as the default value of **ALL** Dict instances?